assets/src/block-validation/components/with-amp-toolbar-button/test/add-toolbar-button-to-block.js

```diff
@@ -38,39 +38,18 @@ describe('withAMPToolbarButton: filtering with errors', () => {
 		block = createBlock(TEST_BLOCK, {});
 		dispatch('core/block-editor').insertBlock(block);
 
-		createStore({
-			reviewLink: 'http://review-link.test',
-			unreviewedValidationErrors: [
-				{
-					clientId: block.clientId,
-					code: 'DISALLOWED_TAG',
-					status: 3,
-					term_id: 12,
-					title: 'Invalid script: <code>jquery.js</code>',
-					type: 'js_error',
-				},
-			],
-			validationErrors: [
-				{
-					clientId: block.clientId,
-					code: 'DISALLOWED_TAG',
-					status: 3,
-					term_id: 12,
-					title: 'Invalid script: <code>jquery.js</code>',
-					type: 'js_error',
-				},
-			],
-		});
-	});
```

Comment:
Previously we were using `registerStore` to create redux stores and then initializing them at required places. With this previously, we were able to initialize it with the initial state of our choice. 

With the current implementation, we are initializing our store itself where it has been defined so there will be a single instance of store everywhere. We have combined both of them in one because it's the job of the reducer to decide which validation error will be assigned to unreviewed or reviewed as per

https://github.com/ampproject/amp-wp/blob/9f7014a73e3c16adc310df1575e6d8d4a9435e44/assets/src/block-validation/store/index.js#L82-L123

Previously, `unreviewedValidationErrors` was being passed manually(which shouldn't be the case) to check if we are getting the expected results. By passing 1 as status the reducer automatically adds the validation error to the `unreviewedValidationErrors` state.

Also, see how Gutenberg creates stores: https://github.com/WordPress/gutenberg/blob/trunk/packages/notices/src/store/index.js